tools/osbuilder/README.md

```diff
@@ -80,8 +80,8 @@ filesystem components to generate an initrd.
 3. When generating an image, the initrd is extracted to obtain the base rootfs for
 the image.
 
-CentOS is the default distro for building the rootfs, to use a different one, you can set `DISTRO=<your_distro>`.
-For example `make USE_DOCKER=true DISTRO=ubuntu rootfs` will make Ubuntu rootfs using Docker.
+Ubuntu is the default distro for building the rootfs, to use a different one, you can set `DISTRO=alpine|clearlinux|debian|ubuntu`.
```

Comment:
Speaking of defaults you will want to sync/update the values from https://github.com/kata-containers/kata-containers/blob/main/versions.yaml#L118-L132

Reply:
@fidencio https://github.com/kata-containers/kata-containers/pull/3328#issuecomment-999322121

Reply:
Thanks, fixed!